module1/src/App.js

```diff
@@ -0,0 +1,35 @@
+import { Component } from "react";
+import "./css/App.css";
+import Counter from "./components/Counter";
+import SearchForm from "./components/SearchForm";
+import GenreSelect from "./components/GenreSelect";
+
+class App extends Component {
+  handleSearch = (query) => {
+    console.log("Searching:", query);
+  };
+
+  handleSelect = (genre) => {
+    console.log("Selected genre:", genre);
```

Comment:
onSelect you should change style for genre button, console.log is  incorrect here